frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.jsx

```diff
@@ -158,12 +166,27 @@ function ExpressionEditorSuggestionsListItem({
             data-ignore-outside-clicks
           />
         </SuggestionTitle>
-        <QueryColumnInfoIcon
-          query={query}
-          stageIndex={stageIndex}
-          column={suggestion.column}
-          position="right"
-        />
+        {helpText && (
```

Comment:
When the popover is shown on hover:
- There's a link in the popover but it's not possible to open it (see video)
- It's not possible to select the text in the popover when it's shown on hover
- It's not possible to use the right-click inside the popover

Instead the function gets selected and added to the expression.


https://github.com/metabase/metabase/assets/6830683/911c5738-a539-4a4b-8b89-22d5df1abac6

Reply:
Good catch. I've fixed the issue, except for selecting text, since in the default popover this is also not possible.

Reply:
The popover is still getting closed when I middle-click the link.